tests/integration_tests/config/test_config.py

```diff
@@ -253,22 +250,6 @@ def test_config_documents(client: TestClient, data_db: Session, user_header_toke
     assert "Translation" in data["document"]["variants"]
 
 
-def test_config_events(client: TestClient, data_db: Session, user_header_token):
```

Comment:
Why was this removed? Granted its not up to much, is it sufficiently covered elsewhere?

Reply:
Because the `events` key is no longer part of the config  - we've removed it because we've pushed it into the taxonomy

Reply:
The events key had a single key-value pair as a value: 
```
"events": {
  "event_types": {
    "Amended": "Amended Event", ...
  }
}
```